lib/ansible/modules/find.py

```diff
@@ -515,7 +515,7 @@ def main():
     skipped = {}
 
     def handle_walk_errors(e):
-        if e.errno in (errno.EPERM, errno.EACCES):
+        if e.errno in (errno.EPERM, errno.EACCES, errno.ENOENT):
```

Comment:
Which OSErrors need to be fatal? Since the errors suppressed here are still a warning, I'm curious if we can handle all OSError exceptions. LGTM otherwise.

Reply:
Ok, I guess this is unclear, but there may be some.

Reply:
most OS errors at this point would be about resource availability: no fs, mem, etc
The IO errors that are recoverable/ignorable should be those covered: mostly permission based or the new one 'file disappeared!', which I would make fatal in most cases but seems fine to just issue a warning for a `find`.

Reply:
@bcoca Thanks for clarifying, that makes sense. I was confused about some of the exception types from the Python documentation alone, and mistakenly thought os.scandir() might raise fatal non-OSErrors (like MemoryError) that os.walk would not ignore. Now I'm amazed os.walk() ignores *all* OSErrors by default.